data_services/deploy/ansible/roles/forecastio/files/data_queues/forecastio/forecastio_air.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
```

Comment:
I think what would make the most sense would be to make an ansible role inside the new repository that we can use inside this playbook.  For that, we would probably move much (if not all) of this code into the new repo.